client/landing/gutenboarding/components/close-button/index.tsx

```diff
@@ -5,14 +5,14 @@ import React from 'react';
 import { Button } from '@wordpress/components';
 import { useI18n } from '@automattic/react-i18n';
 
-interface CloseButtonProps {
+interface CloseButtonProps extends Button.ButtonProps {
 	onClose: () => void;
 }
 
-const CloseButton = ( { onClose }: CloseButtonProps ) => {
+const CloseButton = ( { onClose, ...buttonProps }: CloseButtonProps ) => {
```

Comment:
Generally I prefer to type these as:

```suggestion
const CloseButton: React.FunctionComponent< CloseButtonProps > = ( { onClose, ...buttonProps } ) => {
```

Reply:
Suggestion commited.